File: AgentieTursimCS/Repository/Interfaces/IRepoAngajat.cs
#nullable enable
using System;
using AgentieTurismCS.Domain;

namespace AgentieTurismCS.Repository.Interfaces
{
    public interface IRepoAngajat : IRepo<Guid, Angajat>
    {
        public Angajat? FindAngajatByUsername(string username);
    }
}

File: SocketsJava/Persistence/Interfaces/IRepoAngajat.java
package agentie.persistence.Interfaces;

import agentie.model.Angajat;
import java.util.Optional;
import java.util.UUID;

public interface IRepoAngajat extends IRepo<UUID, Angajat> {
    Optional<Angajat> findOneByUsername(String username);
}
